Pull Request for review

visva-dev opened the pull request:
## Microverse 7th project - Using Bootstrap
- This project requests students to build a replica of the news site Newsweek.com using the Bootstrap framework.
- Aligned all elements with Bootstrap and CSS.

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+  margin-top: -15px;
+}
+
+.see-all {
+  margin-top: -1.3rem !important;
+  margin-left: -1.3rem !important;
+  margin-right: -1.3rem !important;
+  padding: 1rem;
+  margin-bottom: -5px !important;
+}
```
Please try to minimize use of `!important`

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+
+.periscope-culture {
+  margin-top: 1.5rem;
+  margin-left: -0.6rem !important;
+  margin-right: -0.6rem !important;
```
Please try to minimize use of `!important`

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+  width: auto;
+}
+
+.editors-pick {
+  margin-right: -1.5rem !important;
+}
+
+.editors2 {
+  margin-right: -3rem !important;
+  margin-top: 2rem !important;
+}
```
Please try to minimize use of `!important`

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+  top: 18px;
+}
+
+.in-depth {
+  margin-left: -6rem !important;
+}
+
+.text-position {
+  margin-left: -1.1rem !important;
```
Please try to minimize use of `!important`

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+  .category-7 {
+    font: 'Roboto Condensed', sans-serif;
+    color: #f72210;
+    text-transform: uppercase;
```
[OPTIONAL]- You can use the bootstrap class `text-uppercase` for this. Feel free to use [this cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information.

BrittanyBlake commented on styles.css:
```diff
@@ -0,0 +1,158 @@
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  text-transform: uppercase;
```
[OPTIONAL]- You can use the bootstrap class `text-uppercase` for this. Feel free to use [this cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information.

visva-dev commented:
- Fixed navbar extends to the whole page.

visva-dev commented:
## Navigation Bar
- Fixed the navbar extends to the whole page
- Aligned search icon properly in the nav links
- Made nav bottom links to disappear until the large screen
## Main content
- Fixed horizontal scroll on md, lg screen sizes & too much white space problems
- Fixed styling of Debate and Opinion sections on md & sm screens
- Fixed editors section horizontal scroll & images stretch & title wrap on md screen
- Fixed featured slideshows section alignment on sm and md screens sizes
- Updated category styling
- Reduced using of !Important 
## Footer
- Fixed subscribe and newsletter section responsiveness

rammazzoti2000 commented on styles.css:
```diff
@@ -0,0 +1,209 @@
+  .category {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 12.2em;
+  }
+
+  .category-1 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 12.2em;
+  }
+
+  .category-2 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 7.7em;
+  }
+
+  .category-3 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-5 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-6 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 5.6em;
+  }
+
+  .category-7 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-8 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+}
```
No repeating code, you can notice all this classes are using the same font-family and most of them the same font-size, please refactor that.

rammazzoti2000 commented on styles.css:
```diff
@@ -0,0 +1,209 @@
+    top: 3.8rem;
+  }
+
+  .category {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 13.2em;
+  }
+
+  .category-1 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 18.2em;
+  }
+
+  .category-2 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 13em;
+  }
+
+  .category-3 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 9.2em;
+  }
+
+  .category-5 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 18.2em;
+  }
+
+  .category-6 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.6em;
+  }
+
+  .category-7 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 15.2em;
+  }
+}
```
Same as above

rammazzoti2000 commented on styles.css:
```diff
@@ -0,0 +1,209 @@
+  bottom: 10.7em;
+}
+
+.category-2 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-3 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 9.2em;
+}
+
+.category-4 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-5 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.category-6 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 5.9em;
+}
+
+.category-7 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.vs-img {
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  bottom: -20px;
+  z-index: 1;
+}
```
No repeating code, you can notice all this classes are using the same font-family and most of them the same font-size, please refactor that.

visva-dev commented:
## Issues fixed 
- Fixed bottom-nav styling for all screen sizes
- Fixed featured stories category styling
- Fixed all categories not sticking to the bottom problem
- Fixed More stories section alignment on all screens
- Updated responsiveness of debate and opinion sections
- Fixed right side articles styling
- Menu and subscribe sections updated style
- Fixed footer positioning
- Updated order
- Cleaned syntax with no repeating code approach

visva-dev commented:
## Changes made
- Fixed syntax errors
- Updated positioning of subscribe section
- Updated live dome link in the readme file
